config.go

```diff
@@ -54,11 +54,10 @@ type PumpConfig struct {
 	// }
 	// ```
 	Filters analytics.AnalyticsFilters `json:"filters"`
-	// You can configure a different timeout for each pump with the configuration option `timeout`.
-	// Its default value is `0` seconds, which means that the pump will wait for the writing
-	// operation forever.
+	// By default, a pump will wait forever for each write operation to complete; you can configure an optional timeout by setting the configuration option timeout.
```

Comment:
Could you please put the last _timeout_ in `code case`, as it's the name of the key?